docs/configuration/SUMMA_model_decisions.md

```diff
@@ -3,3 +3,396 @@
+
+<a id="stomResist"></a>
+##  6. stomResist
+Function for stomatal resistance
+
+| Option | Description |
+|---|---|
+| BallBerry | **TODO: Describe BallBerry <br> [Reference](http://doi.org/)** |
+| Jarvis | **TODO: Describe Jarvis <br> [Reference](http://doi.org/)** |
+| simpleResistance | **TODO: Describe simpleResistance <br> [Reference](http://doi.org/)** |
+| BallBerryFlex | **TODO: Describe BallBerryFlex <br> [Reference](http://doi.org/)** |
+| BallBerryTest | **TODO: Describe BallBerryTest <br> [Reference](http://doi.org/)** |
+
+<a id="bbTempFunc"></a>
+##  7. bbTempFunc
+Ball-Berry: leaf temperature controls on photosynthesis + stomatal resistance
+
+| Option | Description |
+|---|---|
+| q10Func | **TODO: Describe q10Func <br> [Reference](http://doi.org/)** |
+| Arrhenius | **TODO: Describe Arrhenius <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbHumdFunc"></a>
+##  8. bbHumdFunc
+Ball-Berry: humidity controls on stomatal resistance
+
+| Option | Description |
+|---|---|
+| humidLeafSurface | **TODO: Describe humidLeafSurface <br> [Reference](http://doi.org/)** |
+| scaledHyperbolic | **TODO: Describe scaledHyperbolic <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbElecFunc"></a>
+## 9. bbElecFunc
+Ball-Berry: dependence of photosynthesis on PAR
+
+| Option | Description |
+|---|---|
+| linear | **TODO: Describe linear <br> [Reference](http://doi.org/)** |
+| linearJmax | **TODO: Describe linearJmax <br> [Reference](http://doi.org/)** |
+| quadraticJmax | **TODO: Describe quadraticJmax <br> [Reference](http://doi.org/)** |
+
+<a id="bbCO2point"></a>
+## 10. bbCO2point
+Ball-Berry: use of CO2 compensation point to calculate stomatal resistance
+
+| Option | Description |
+|---|---|
+| origBWB | **TODO: Describe origBWB <br> [Reference](http://doi.org/)** |
+| Leuning | **TODO: Describe Leuning <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbNumerics"></a>
+## 11. bbNumerics
+Ball-Berry: iterative numerical solution method
+
+| Option | Description |
+|---|---|
+| NoahMPsolution | **TODO: Describe NoahMPsolution <br> [Reference](http://doi.org/)** |
+| newtonRaphson | **TODO: Describe newtonRaphson <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbAssimFnc"></a>
+## 12. bbAssimFnc
+Ball-Berry: controls on carbon assimilation
+
+| Option | Description |
+|---|---|
+| colimitation | **TODO: Describe colimitation <br> [Reference](http://doi.org/)** |
+| minFunc | **TODO: Describe minFunc <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbCanIntg8"></a>
+## 13. bbCanIntg8
+Ball-Berry: scaling of photosynthesis from the leaf to the canopy
+
+| Option | Description |
+|---|---|
+| constantScaling | **TODO: Describe constantScaling <br> [Reference](http://doi.org/)** |
+| laiScaling | **TODO: Describe laiScaling <br> [Reference](http://doi.org/)** |
+
+
+<a id="num_method"></a>
+## 14. num_method
+Numerical method
```

Comment:
Might be helpful to say what this numerical method is solving (as opposed to `bbNumerics` or similar)

Reply:
See above - fully agreed. Should be addressed as part of #325